sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -96,7 +108,7 @@ export class EducationProgramController {
     @Body() payload: EducationProgramDto,
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<number> {
-    const newProgram = await this.saveProgram(userToken, payload);
+    const newProgram = await this.saveProgram(userToken, payload, undefined);
```

Comment:
Is undefined passed here with intention of saving null in the programId column in the database ?

Reply:
yes, it was. functions have been changed, so `saveProgram` now has only 2 parameters